website_cookie_notice/views/website.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+<data>
+
+<template id="loader" name="Cookies script loader">
+    <script type="text/javascript"
+        src="/website_cookie_notice/static/src/js/accept_cookies.js">
+    </script>
+</template>
+
+<template id="message" name="Cookies notice">
+    <div t-if="request and
+        not request.httpsession.get('accepted_cookies')"
+        id="website_cookie_notice"
+        class="container-fluid bg-warning cc-cookies">
+        <div class="container">
+            <div class="row">
+                <div class="col-xs-10">We use cookies in this website. Read about them in our <a href="/page/privacy">privacy policy</a>. To disable them, configure your browser properly. If you keep using this website, you are accepting those.</div>
+                <div class="col-xs-2 text-center">
```

Comment:
@lasley I can't see what it would solve. I tried 
```
                <div class="col-xs-1 col-xs-offset-1 text-center">
                    <a class="btn btn-primary" href="#">OK</a>
                </div>
```
but it has problems:
![cookie](https://cloud.githubusercontent.com/assets/1033131/15006908/9dd860d8-11d4-11e6-85bb-2173fbd4cacf.png)